src/ContaoManager/Plugin.php

```diff
@@ -13,6 +13,7 @@
 use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
 use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
 use HeimrichHannot\GoogleMapsBundle\HeimrichHannotGoogleMapsBundle;
+use Hofff\Contao\Consent\Bridge\HofffContaoConsentBridgeBundle;
```

Comment:
Won't that throw an exception if the bundle is not installed? Did you test that case sufficiently?

Reply:
No, you can use `::class` for classes that does not exist, see https://www.php.net/manual/de/language.oop5.constants.php#121613 or https://3v4l.org/REY04#v7.0.33

Reply:
OK, then I guess this is also true if you use a `use` statement instead of the fully qualified class name insteadt?

Reply:
Yes, it only imports the name and does not try to autoload the class. See https://3v4l.org/erFYN